set-link/entrypoint.sh

```diff
@@ -84,35 +90,36 @@ echo "New title with removed ticket is '${new_title}'"
 
 # Add the story number to the PR title if it isn't already there
 if [[ "$story_removed_title" != *"[sc-$story]"* ]]; then
-    new_title="[sc-${story}] ${story_removed_title^}"
+  new_title="[sc-${story}] ${story_removed_title^}"
 fi
 
 echo "Final new title is '${new_title}'"
 
-cat > ~/.netrc <<-EOF
-machine api.github.com
-    login $GITHUB_ACTOR
-    password $GITHUB_TOKEN
-EOF
+SKIP_COMMENT="0"
+"$OK" -j list_issue_comments "$GITHUB_REPOSITORY" "$number" | jq -e '. | map(select(.user?.login? == "shortcut-integration[bot]")) | .[0]' >/dev/null || SKIP_COMMENT="$?"
 
-if [[ "$story" != "" && ( "$body" != "$new_body" || "$title" != "$new_title" ) ]]; then
-    args=("title='$new_title'")
-    if [[ "$COMMENT_ONLY" != "1" && "$SKIP_LINK" != "1" ]]; then
-        args+=("body='$new_body'")
-    fi
+if [[ "$story" != "" ]] && { [[ "$body" != "$new_body" || "$title" != "$new_title" ]]; }; then
+  args=()
+  [[ "$title" != "$new_title" ]] && args+=("title='$new_title'")
```

Comment:
Interestingly this is still being added despite the if statement and the title/new_title being the same. Not really sure why.

Reply:
Lol, the story of this script 😆

Reply:
@hoshsadiq Is it possible there's an extra space in one of them? This script may be adding a space at the beginning sometimes... I was noticing that sometimes when I was fumbling with this script, but never got around to actually looking at it.

Reply:
Cool, I've updated it to have address that.